README.md

```diff
@@ -6,3 +6,5 @@ A microservice which calculates and makes available the Complexity Of Need assoc
 Owned by the Manage POM Cases Team - mailto:elephants@digital.justice.gov.uk 
 
 API Specification can be viewed [Here](https://editor.swagger.io/?url=https://raw.githack.com/ministryofjustice/hmpps-complexity-of-need/main/Complexity%20Of%20Need%20API%20Specification.yaml)
+
+Posted Event Specification can by viewed [Here](https://playground.asyncapi.io/?load=https://raw.githack.com/ministryofjustice/hmpps-complexity-of-need/main/Complexity%20Of%20Need%20Event%20Specification.yaml)
```

Comment:
This link doesn't work - it seens you to a 'Streetlights' (which I presume is a demo)

Reply:
Yes - a bit of a loop here - because the linl wont work until this pull request has been put into main. Maybe should have been two pull requests - one to include the specification file, then a second to reference the file in the readme.

Reply:
Whilst it's in this PR/branch, we can view it here:

https://playground.asyncapi.io/?load=https://raw.githack.com/ministryofjustice/hmpps-complexity-of-need/MO539_-_Event_Documentation/Complexity%20of%20Need%20Event%20Specification.yaml

Reply:
Thanks Ollie - thats the correct output - is that sufficient for approval of the pull request?